src/animation/AnimationClip.js

```diff
@@ -420,6 +420,45 @@ Object.assign( AnimationClip.prototype, {
 		return this;
 
 	},
+	
+	slice: function(from, to, name) {
```

Comment:
Code style: Extra spaces are necessary after opening bracket and before closing bracket `slice: function( from, to, name ) {`

https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2#blocks